.gitignore

```diff
@@ -0,0 +1,3 @@
+node_modules/
```

Comment:
- [ ] Please next time when you request a code review ensure to use the same PR for that and do not open a new PR.